scarpe.gemspec

```diff
@@ -31,7 +31,9 @@ Gem::Specification.new do |spec|
   spec.require_paths = ["lib"]
 
   # Uncomment to register a new dependency of your gem
+  spec.add_dependency "fastimage"
```

Comment:
I'm open to adding new runtime dependencies if they provide good value for the repo, but would need a good argument behind adding it. (example glimmer-dsl-libui is needed to run the glimmer display service altogether)

Reply:
This is a genuinely interesting case. A dependency to get image size from a URL makes sense -- we don't want to write and maintain that. And we kind of need it because the information flows back to the Ruby side. We *could* get it from Webview, but then it's not at all clear how we test with no display lib, for instance. Right now, Webview still isn't structural, and it would be nice for it to remain that way.

Reply:
that's a good point! and fastimage looks legit, happy to keep it!